packages/manager/modules/sms/src/sms/order/telecom-sms-order-service.html

```diff
@@ -4,171 +4,205 @@
     </div>
 
     <div class="widget-presentation" data-ng-if="!$ctrl.loading.init">
-        <header class="widget-presentation-header">
-            <h2
-                class="widget-presentation-title"
-                data-translate="sms_order_title"
-            ></h2>
-        </header>
+        <oui-header data-heading="{{ 'sms_order_title' | translate }}">
+        </oui-header>
 
         <form id="$ctrl.orm" name="$ctrl.orm">
             <div class="row">
-                <div class="col-md-6">
-                    <!-- ACCOUNT -->
-                    <div class="form-group">
+                <!-- ACCOUNT -->
+                <div class="col-sm-12">
```

Comment:
i guess we can consider to use `col-12`.
Also, what about for tablets and laptop devices ?

Reply:
The class "col-12" doesn't works.
The display is responsive for PC and tablets / smartphone.